docs/howto/installprotege.md

```diff
@@ -0,0 +1,6 @@
+# Installing Protege
+
+1. Follow the instructions on the GO wiki page: [http://wiki.geneontology.org/index.php/Protege_setup_for_GO_Eds](http://wiki.geneontology.org/index.php/Protege_setup_for_GO_Eds)
+
+2. _Need to add more here about the different Views and Tabs needed for working._
```

Comment:
I'm not sure what this means

Reply:
Looks like this is  one of a number of docs that are just outdated versions of docs we already have.  I listed the ones I think we need on this ticket: https://github.com/OBOAcademy/obook/issues/92.  I think the reset (including this) should just be deleted).

Reply:
removed it